cmd/kubectl-gadget/sigsnoop.go

```diff
@@ -0,0 +1,226 @@
+//go:build !windows
+// +build !windows
+
```

Comment:
Why not windows?

Reply:
I am using a `unix` package to get the signal name and this package does not exist on Windows, hence removing this OS from the build.

Reply:
hmm, I see. Let' keep going this way, we can try later on to find a way for supporting this on windows too.

Reply:
Thinking about this, I really think this is a not big problem not have this gadget compiled for Windows.
Indeed, all other gadgets are compiled for Windows, but the eBPF code traces Linux kernel function, so they will run but no events will be printed.

Particularly for this one, I think we will never be able to port it to Windows because Windows does not offer the POSIX signal API.

Reply:
I didn't mean to support tracing Windows (yet). This code is part of the client, the use case here is to execute kubectl gadget on a Windows machine tracing a Kubernetes cluster running in Linux. 

A way to remove this limitation could be to move the logic that uses `"golang.org/x/sys/unix"` to the tracer, i.e, the client will send the text introduced by the user without any parsing and the server will perform the SIGXXX to the signal number conversion. What do you think?

Reply:
> the use case here is to execute kubectl gadget on a Windows machine tracing a Kubernetes cluster running in Linux.

I did not think to this use case, seems legit in this case.

> A way to remove this limitation could be to move the logic that uses "golang.org/x/sys/unix" to the tracer, i.e, the client will send the text introduced by the user without any parsing and the server will perform the SIGXXX to the signal number conversion. What do you think?

I think we do not have other choices since we cannot do something like this in golang:

```C
#ifdef OS_LINUX
...
#endif
```